interchain-exchange/tutorial/04-order-pairs.md

```diff
@@ -65,7 +65,7 @@ When a packet with an order book creation is received, the validity of the trans
 If the pair does not exist yet, it can be added to the keeper.
```

Comment:
```suggestion
If the pair does not exist, you can add it to the keeper.
```

Reply:
- we don't need "yet" after exist 
- can we use imperative language and have the user add the pair to the keeper?